lib/pfcp/src/pfcp_xact.c

```diff
@@ -444,6 +442,10 @@ Status PfcpXactUpdateRx(PfcpXact *xact, uint8_t type) {
                             "local " : "remote", xact->step, type,
                             GetIP(&xact->gnode->sock->remoteAddr),
                             GetPort(&xact->gnode->sock->remoteAddr));
+
```

Comment:
You moved `TimerStop` into **PFCP_XACT_FINAL_STAGE** of case **PFCP_LOCAL_ORIGINATOR**.
In this case,
If the pfcp xact belongs **PFCP_REMOTE_ORIGINATOR**, I suppose its timer won't be stopped correctly.

If my assumption is wrong, please let me know.

Tks!

Reply:
Sorry for late reply.

`xact->timerResponse`  exists only when `xact->origin` is **PFCP_LOCAL_ORIGINATOR** but does not exist when **PFCP_REMOTE_ORIGINATOR**.

`TimerStop()` should be moved here, because otherwise `xact->timerResponse` will be stopped right after it is started.

Thanks,